wikicharts/content_gap_plots.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.        chart.format(title='Articles about underrepresented regions', data_source="Knowledge Gap Index", titlepad=15) 
</pre><p>This is a broader question which I'll bring up on the team channel, but we really need to be careful to use accurate names for these metrics (<em>I </em>was getting quite confused about what they actually meant, and it's only resolved now that I've spent a lot of time with the code).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>